tools/dimspy/align_samples.xml

```diff
@@ -29,6 +31,9 @@
         <param name="ppm" argument="--ppm" type="float" value="2.0" label="Ppm error tolerance" help="Maximum tolerated m/z deviation across samples in parts per million (ppm)." />
         <param name="delimiter" argument="--delimiter" type="hidden" value="tab" />
         <expand macro="hdf5_pm_to_txt" />
+        <section name="adv" title="Advanced options" expanded="False">
```

Comment:
a hidden section with one parameter, is just increasing complexity and does not hide much, isn't it?

Reply:
Noted - more advanced parameters will be added in the next version - I would like to keep this as a place holder.